pkgs/package-overrides.nix

```diff
@@ -210,12 +242,12 @@ in
     });
 
     json =
-      if lib.versionOlder prev.php.version "8.0" then
+      if lib.versionAtLeast prev.php.version "8.0" then
+        throw "php.extensions.json requires PHP version < 8.0."
```

Comment:
`php.extensions.json is built into PHP ≥ 8.0`

Reply:
I will amend the message. Thanks.